tests/acceptance/features/cliPasswordExpire/occExpireUserPassword.feature

```diff
@@ -8,31 +8,32 @@ Feature: expire user's password using the occ command
   Background:
     Given user "Alice" has been created with default attributes and small skeleton files
 
+
   Scenario: admin tries to expire user password without setting expiration rule
     When the administrator expires the password of user "Alice" using the occ command
     Then the command should have failed with exit code 1
     And the command output should contain the text 'Cannot use this command because no expiration rule was configured'
 
+
   Scenario: admin expires user password after setting expiration rule
     Given the administrator has enabled the days until user password expires user password policy
     When the administrator expires the password of user "Alice" using the occ command
     Then the command should have been successful
     And the command output should contain the text 'The password for Alice is set to expire'
 
-  @issue-313
+
   Scenario: admin expires password of a user and then the user downloads a file
     Given the administrator has enabled the days until user password expires user password policy
     When the administrator expires the password of user "Alice" using the occ command
-    Then the downloaded content when downloading file "/textfile0.txt" for user "Alice" with range "bytes=0-1" should be "ow"
-#    Then user "Alice" using password "%regular%" should not be able to download file "textfile0.txt"
+    Then user "Alice" downloading the file "textfile0.txt" and using the password "%regular%" should receive hint to change their password
+
 
-  @issue-313
   Scenario: admin expires password of a user and then the user deletes a file
     Given the administrator has enabled the days until user password expires user password policy
     And the administrator has expired the password of user "Alice"
     When user "Alice" deletes file "/textfile0.txt" using the WebDAV API
-    Then as "Alice" file "/textfile0.txt" should not exist
-#    Then as "Alice" file "/textfile0.txt" should exist
+    Then the HTTP status code should be "503"
```

Comment:
Shouldn't this be a 401 unauthorized, as "Alice" should not be able to login?

Reply:
I agree, it would be the preferred status code. However, the error comes form the underlying webdav implementation which we don't want to touch. The error is `503` either way, so we're not really leaking any information about whether the users exists or not.

Reply:
Ahh okay, makes sense thanks for explaining it =)